doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -219,10 +225,10 @@ replace terms with related ones only in contexts that are syntactic
 compositions of parametric morphism instances declared with the
 following command.
 
-.. cmd:: Add Parametric Morphism {* @binder } : (@ident {+ @term__1}) with signature @term__2 as @ident
+.. cmd:: Add Parametric Morphism {* @binder } : @one_term with signature @term as @ident
```

Comment:
Is it really `@term` and not `@one_term`?

Reply:
Yes, it is.  From `g_rewrite.mlg`:

```
| #[ atts = rewrite_attributes; ] ![ open_proof ] [ "Add" "Parametric" "Morphism"
        binders(binders) ":" constr(m)
        "with" "signature" lconstr(s) "as" ident(n) ]
```